app/serializers/todo_serializer.rb

```diff
@@ -1,9 +1,11 @@
 # frozen_string_literal: true
 
 class TodoSerializer < ActiveModel::Serializer
+  include Rails.application.routes.url_helpers
+
   attributes :id, :title, :url, :order, :completed
 
   def url
-    "#{ENV.fetch('HOST_URL')}/#{object.id}"
+    "#{ENV.fetch('HOST_URL')}#{todos_path}/#{object.id}"
```

Comment:
Let's build the URL, to ensure it's properly formatted.

Reply:
e30bc71d2ced90fc64cc86a24c7e009c647079c0